holoviews/element/raster.py

```diff
@@ -644,6 +421,8 @@ class RGB(Image):
         If an alpha channel is supplied, the defined alpha_dimension
         is automatically appended to this list.""")
 
+    _vdim_reductions = {1: Image}
```

Comment:
Ah yes, I remember the issue. I don't know if you get a reference between issues/PRs from the comments but we can find out: I believe the relevant issue is #1173